src/flask_ml/flask_ml_server/MLServer.py

```diff
@@ -4,7 +4,7 @@
 from flask import Flask, jsonify, request
 from pydantic import BaseModel, ValidationError
 
-from .models import ErrorResponseModel, RequestModel, ResponseModel
+from .models import ErrorResponseModel, RequestModel, ResponseModel, create_flask_response

```

Comment:
create_flask_response is a pure function that takes a pydantic model and creates a FlaskML response. This is to decouple pydantic and FlaskML.

Reply:
Great idea!